src/app/components/MediaLoader/index.tsx

```diff
@@ -238,7 +238,11 @@ const MediaLoader = ({ blocks, className, embedded }: Props) => {
         )
       }
       {showPortraitTitle && (
-        <strong css={styles.titlePortrait}>Watch Moments</strong>
+        <strong css={styles.titlePortrait}>
+          {translations.media.watchMoments
+            ? translations.media.watchMoments
+            : 'Watch Moments'}
```

Comment:
Could this be simplified: 
```suggestion
          {translations.media.watchMoments ||  'Watch Moments' }
```